plugin/trino-ranger/README.md

```diff
@@ -0,0 +1,25 @@
+# Apache Ranger authorizer for Trino
```

Comment:
I think this readme file is not needed, let's make sure docs cover things like that.

Reply:
Done.